resources/assets/js/components/DetailProject.js

```diff
@@ -0,0 +1,139 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import { Link } from 'react-router';
+import TableMemberAssign from './TableMemberAssign';
+import MyGlobleSetting from './MyGlobleSetting';
+class DetailProject extends Component {
+  constructor(props) {
+   super(props);
+   this.state = {value: '', project: '', member_roles: '', messages: ''};
+ }
+ componentDidMount(){
+  axios.get(MyGlobleSetting.url + `/api/projects/detail/${this.props.params.id}`)
+  .then(response => {
+    this.setState({ project: response.data.project, member_roles: response.data.member_roles, messages: response.data.messages });
+  })
+  .catch(function (error) {
+    console.log(error);
+  })
+ }
+  projectStatus(status)
+  {
+    switch(status) {
+      case '1': {
+        return 'planend';
+        break;
```

Comment:
The previous line is a `return` statement, so you don't need to use `break`.

Reply:
i have do it